client/my-sites/site-settings/form-general.jsx

```diff
@@ -282,19 +283,23 @@ export class SiteSettingsFormGeneral extends Component {
 		const {
 			fields,
 			handleRadio,
+			updateFields,
 			isRequestingSettings,
 			eventTracker,
 			siteIsJetpack,
+			trackEvent,
 			translate,
 		} = this.props;
 
+		const currentValue = parseInt( fields.blog_public, 10 );
+
 		return (
 			<FormFieldset>
-				<FormLabel>
+				<FormLabel className="site-settings__visibility-label">
```

Comment:
> Perhaps we can remove the radio input when there's no option to make a site private

Maybe something like this (for now):

```suggestion
				{ ! siteIsJetpack && <FormLabel className="site-settings__visibility-label">
```